File: KevinRene/src/ql/ast/expression/relational/And.java
package ql.ast.expression.relational;

import ql.ast.Expression;
import ql.ast.expression.Binary;
import ql.ast.expression.QLType;
import ql.ast.expression.type.QLBoolean;
import ql.ast.visitor.Visitor;

public class And extends Binary {	
	public And(Expression left, Expression right) {
		super(left, right, "&&");
	}
	
	@Override
	public <T> T accept(Visitor<T> visitor) {		
		return visitor.visit(this);
	}

	@Override
	public QLType getType() {
		return new QLBoolean();
	}
}

File: FelipezConde/QuestionnaireLanguage/AST/Nodes/Expressions/Binaries/And.cs

namespace AST.Nodes.Expressions.Binaries
{
    public class And : Binary
    {
        public And(Expression left, Expression right, PositionInText position)
            : base(left, right, position)
        {}

        public override string ToString()
        {
            return "&&";
        }

        public override T Accept<T>(VisitorInterfaces.IExpressionVisitor<T> visitor)
        {
            return visitor.Visit(this);
        }
    }
}
